accounts-kethereum/src/main/java/pm/gnosis/svalinn/accounts/utils/TransactionExtensions.kt

```diff
@@ -35,8 +35,8 @@ fun Transaction.rlp(signature: ECDSASignature? = null): ByteArray {
 fun Transaction.hash(ecdsaSignature: ECDSASignature? = null) = rlp(ecdsaSignature).let { Sha3Utils.keccak(it) }
 
 private fun Transaction.adjustV(v: Byte): Byte {
-    if (chainId > 0) {
-        return (v.toInt() + 8 + 2 * chainId).toByte()
+    if (chainId > BigInteger.ZERO) {
+        return (chainId.multiply(BigInteger.valueOf((v.toLong() + 8 + 2)))).toByte()
```

Comment:
chainId.multiply(BigInteger.valueOf(2)).plus(BigInteger.valueOf(v.toLong() + 8)).toByte() ?

Reply:
Fixed in 76eedbdddc47d57c5b7388ed6744a98938d5fb7f